Diagnostics/HealthChecker/Analyzer/Security/Invoke-AnalyzerSecurityAMSIConfigState.ps1

```diff
@@ -0,0 +1,71 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+Function Invoke-AnalyzerSecurityAMSIConfigState {
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory = $true)]
+        [ref]$AnalyzeResults,
+
+        [Parameter(Mandatory = $true)]
+        [object]$HealthServerObject,
+
+        [Parameter(Mandatory = $true)]
+        [object]$DisplayGroupingKey
+    )
+
+    Write-Verbose "Calling: $($MyInvocation.MyCommand)"
+    $amsiInformation = $HealthServerObject.ExchangeInformation.AMSIConfiguration
```

Comment:
This should move inside the if statement of taking the action. 

Also add `$amsiWriteType = "Yellow"` here and remove the additional lines of where it is set to Yellow.

Reply:
Done. Will be addressed with the next commit.